query-engine/connectors/sql-query-connector/src/database/transaction.rs

```diff
@@ -152,7 +152,7 @@ impl<'tx> WriteOperations for SqlConnectorTransaction<'tx> {
         trace_id: Option<String>,
     ) -> connector::Result<SelectionResult> {
         catch(self.connection_info.clone(), async move {
-            write::create_record(&self.inner, model, args, trace_id).await
+            write::create_record(&self.inner, self.connection_info.clone(), model, args, trace_id).await
```

Comment:
I don't love cloning and passing the the whole connection_info struct through all these functions. 
Do we really need the whole struct. Could we rather just send down the SqlFamily info.

Reply:
Yep, me neither, I will change this to a nicer way using borrowing, after all, we need this only once.